src/routes/PostDetail/Comments/Comment/Comment.js

```diff
@@ -142,7 +149,7 @@ export default class CommentWithReplies extends Component {
       replying: true,
       triggerReplyAction: true,
       prefillEditor: toMember
-        ? `<p><a data-entity-type="mention" data-user-id="${toMember.id}">${toMember.name}</a> </p>`
+        ? `<p>${TextHelpers.mentionHTML(toMember)}&nbsp;</p>`
```

Comment:
For consistency `mentionHTML` and `topicHTML` helpers in shared should always be used to generate the correct HTML if inserting either into the editor or HTML...